aibolit/__main__.py

```diff
@@ -495,16 +509,16 @@ def get_exit_code(results):
     errors_strings = []
     for result_for_file in results:
         results = result_for_file.get('results')
-        errors_string = result_for_file.get('error_string')
-        if not results and not errors_string:
+        ex = result_for_file.get('exception')
+        if not results and not ex:
             perfect_code_number += 1
-        elif not results and errors_string:
-            if 'javalang.parser.JavaSyntaxError' not in errors_string:
-                errors_strings.append(errors_string)
+        elif not results and ex:
+            if not isinstance(ex, javalang.parser.JavaSyntaxError):
```

Comment:
could you import just `JavaSyntaxError` ?

Reply:
ok but there is no difference. If we import it it won't become just JavaSyntaxError. Import is just an acronym. It will still be javalang.parser.JavaSyntaxError